dts/bindings/video/video-interfaces.yaml

```
@@ -0,0 +1,133 @@
+  port is present in a device node or there is more than one endpoint at a port, or
+  port node needs to be associated with a selected hardware interface, a common
+  scheme using '#address-cells', '#size-cells' and 'reg' properties is used.
+
+  All 'port' nodes can be grouped under an optional 'ports' node, which allows to
+  specify #address-cells, #size-cells properties independently for the 'port' and
+  'endpoint' nodes. For example:
+
+    video_device {
+      ...
+      ports {
+          #address-cells = <1>;
+          #size-cells = <0>;
+
+          port@0 {
+              ...
+              endpoint@0 { ... };
+              endpoint@1 { ... };
+          };
+          port@1 { ... };
+      };
+    };
+
+  Two 'endpoint' nodes must be linked with each other via their 'remote-endpoint'
+  phandles. However, Zephyr does not allow circular dependency, so direct phandle
+  references are currently not possible. A 'remote-endpoint-label' string is used
+  instead to be able to specify, at least, the label of the peer remote-endpoint.
+
+  This binding contains the most common properties needed to configure an endpoint
+  subnode for data exchange with other device. In most cases, properties at the
+  peer 'endpoint' node will be identical, however they might be different when
+  there is any signal modifications on the bus between two devices, e.g. there are
+  logic signal inverters on the lines.
+
+properties:
+  remote-endpoint-label:
+    required: true
+    type: string
+    description: |
+      Label of the 'remote-endpoint' subnode that interfaces with this endpoint.
+      This property is used as a 'work-around' to be able to declare the remote
+      endpoint and should be replaced by a "remote-endpoint" phandle property when
+      Zephyr devicetree supports circular dependency in the future.
+
+  bus-type:
+    type: string
+    enum:
+      - "parallel"
+      - "mipi-csi2-dphy"
+      - "mipi-csi2-cphy"
+      - "mipi-csi1"
+    description: |
+      Data bus type.
+
+  bus-width:
+    type: int
+    description: |
+      Number of data lines actively used, only valid for parallel busses.
+
+  data-shift:
+    type: int
+    description: |
+      On parallel data busses, if bus-width is used to specify the number of
+      data lines, data-shift can be used to specify which data lines are used,
+      e.g. "bus-width=<8>; data-shift=<2>;" means, that lines 9:2 are used.
+
+  hsync-active:
+    type: string
+    enum:
+      - "low"
+      - "high"
+    description: |
+      Active state of the HSYNC signal
+
+  vsync-active:
+    type: string
+    enum:
+      - "low"
+      - "high"
+    description: |
+      Active state of the VSYNC signal.
+
+  pclk-sample:
+    type: string
+    enum:
+      - "falling"
+      - "rising"
+      - "both"
+    description: |
+      Sample data on falling, rising or both edges of the pixel clock signal.
+
```

How about adding a comment there telling that everything that follows is for MIPI?

```suggestion
    # For MIPI
```

---

yes, done.